PocketKit/Sources/Textile/Views/Tags/TagsList.swift

```diff
@@ -0,0 +1,26 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+public protocol TagsList {
+    func arrangeTags(with tags: [String]) -> [TagType]
+}
+
+public extension TagsList {
+    /// Arranges the tags in a list by sorting with three recent tags and then alphabetizing the rest
+    /// - Parameter tags: list of users tags
+    /// - Returns: a list of tags to display to the user
+    func arrangeTags(with tags: [String]) -> [TagType] {
+        var allTags: [String] = []
+        let fetchedTags = tags.reversed()
+        if fetchedTags.count > 3 {
+            let topRecentTags = Array(fetchedTags)[..<3]
```

Comment:
Can we for sure say we will always be getting 3 tags as recent tags?

If you imagine a new user who just added one tag to an article, how would they get 3?

Reply:
In that case, if tags is less than three, then we won't show the tags as recent tags. They will be displayed in alphabetical order.

Reply:
Alright, fair.